misk-hibernate/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -99,6 +101,9 @@ class HibernateModule(
           metrics = metrics
       )
     }).asSingleton()
+    val dataSourceServiceProvider = getProvider(keyOf<DataSourceService>(qualifier))
+    bind(keyOf<DataSourceConnector>(qualifier)).toProvider(dataSourceServiceProvider).asSingleton()
```

Comment:
neat

Reply:
I’d omit `.asSingleton()` since it already inherits singleton-ness from the DataSourceService